lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -823,6 +825,14 @@ def _download_direct_download_link(self, file_url, copy_file_path):
             with closing(online_file):
                 with open(copy_file_path, 'wb') as output:
                     shutil.copyfileobj(online_file, output)
+            # check first 5 lines of file content
+            with open(copy_file_path) as copied_file:
+                for line in islice(copied_file, 5):
+                    if line.lower().find('html') != -1:
+                        raise ValueError("Undownable File.\n" + 
+                            "Please make sure file is publicly accessible\n" +
+                            "File URL: %s" % file_url)
```

Comment:
OK, I see. Hmm. Even `html` isn't a valid test for all cases since it's optional. 

My gut feel is that we should just let the validator handle this case - is there good reason not to do that? Especially if we have an error message like `File X from url www.google.com is not a valid FASTQ file` the user should have a pretty clear idea of what went wrong.